config/install/dropzonejs.settings.yml

```diff
@@ -1 +1 @@
-tmp_dir: ''
+upload_scheme: temporary
```

Comment:
We are configuring the location where php puts the uploaded file, so I think that this wouldn't make sense if it's set to anything else than temporary. @slashrsm might correct me here, but in HA environments it's important that all servers write to the same location. If drupal's tmp folder location configuration is enough to achieve this goal, your original - non configurable approach - was correct.